gulp-tasks/build.js

```diff
@@ -1,37 +1,55 @@
-var gulp = require('gulp'),
-    minCSS = require('gulp-clean-css'),
-    jshint = require('gulp-jshint'),
-    stylish = require('jshint-stylish'),
-    rev = require('gulp-rev'),
-    uglify = require('gulp-uglify'),
-    del = require('del');
-    usemin = require('gulp-usemin')
+var gulp = require('gulp');
+var useref = require('gulp-useref');
```

Comment:
I want to do this and change the way it is modularised so it is more readable as in method 2 here: http://macr.ae/article/splitting-gulpfile-multiple-files.html
Can I open a new issue for this and work on it after this PR is merged?